dojo/models.py

```diff
@@ -2545,6 +2543,11 @@ class FindingAdmin(admin.ModelAdmin):
     x: "Endpoint: " + str(x.endpoint)
 
 
+class Additional_Vulnerability_Reference(models.Model):
+    finding = models.ForeignKey(Finding, editable=False, on_delete=models.CASCADE)
+    vulnerability_reference = models.TextField(max_length=50, blank=False, null=False)
```

Comment:
We need a type somewhere.

Reply:
Where would the type come from and what would we need it for?

Reply:
filtering and some deduplication logic will be more simple if we have the type of reference.
For ex: "cve" or "ghsa"

Reply:
The type would typically be the prefix of the vulnerability reference itself, at least for all sources I know about. I still think it adds more complexity and redundant data with little benefit.

- **Filtering:** We have 117 `icontains` in `filters.py`, wouldn't this work for the vulnerability references as well? Otherwise we get one more field on some already overcrowded filter forms.
- **Deduplication:** To calculate the `hash_code` we will most likely just concetanate the vulnerability references. How would the type come into play for that?

Reply:
Currently nothing has changed for deduplication, because it still uses the `cve` field. In the next PR I will work on the `hash_code` calculation, we should discuss these changes then.